packages/react/src/components/Authenticator/FederatedIdentity/controls/IdentitiesControl.tsx

```diff
@@ -0,0 +1,10 @@
+import { IdentityControl } from './IdentityControl';
+import { ProviderData } from '../types';
+
+export interface IdentitiesControl<T extends string = string> {
+  (props: {
+    renderListItem?: { (data: ProviderData): React.JSX.Element };
+    children?: React.ReactNode;
```

Comment:
What is the behavior of the UI if i pass both of these as `props`?

Reply:
Good question. I was planning on throwing an error, but now that I think about it a union probably makes more sense.